test/test_datasets.py

```diff
@@ -2168,5 +2168,23 @@ def inject_fake_data(self, tmpdir, config):
         return num_sequences * (num_examples_per_sequence - 1)
 
 
+class EuroSATTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.EuroSAT
+
+    def inject_fake_data(self, tmpdir, config):
+        tmpdir = pathlib.Path(tmpdir)
+
+        wnid = "AnnualCrop"
```

Comment:
"wnid" is specific to `ImageNet` meaning "WordNet ID". I think you mean "category", right?

Reply:
indeed